modules/base/rendering/grids/renderablegrid.cpp

```diff
@@ -188,7 +187,7 @@ RenderableGrid::RenderableGrid(const ghoul::Dictionary& dictionary)
 }
 
 bool RenderableGrid::isReady() const {
-    return _hasLabels ? _gridProgram && _labels->isReady() : _gridProgram != nullptr;
+    return _gridProgram && _hasLabels ? _labels->isReady() : true;
```

Comment:
confused about the order here. May need a parenthesis?

```suggestion
    return _gridProgram && (_hasLabels ? _labels->isReady() : true);
```